lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/exception/package-info.java

```diff
@@ -17,6 +17,6 @@
  * under the License.
  */
 /**
- * Wake exceptions.
+ * Public interfaces and factories for Wake's core API.
  */
-package org.apache.reef.wake.exception;
+package org.apache.reef.wake;
```

Comment:
Agreed I accidentally change wake when I first started adding the new exception. I reverted them.